libmscore/utils.cpp

```diff
@@ -1091,16 +1091,18 @@ InstrumentChange* Score::prevInstrumentChange(Segment* seg, const Part* part, bo
 //    an instrument change.
 //---------------------------------------------------------
 
-InstrumentChange* Score::nextInstrumentChange(Segment* seg, const Staff* staff, bool lookForStaffTypeChange)
+InstrumentChange* Score::nextInstrumentChange(Segment* seg, const Part* part, bool lookForStaffTypeChange)
       {
+      int minTrack = part->staff(0)->idx() * VOICES;
+      int maxTrack = (part->staff(part->nstaves() - 1)->idx() + 1) * VOICES;
       while (seg = seg->next1()) {
```

Comment:
The compiler suggests writing this as `while ((seg = seg->next1()))` instead because it's being used as a truth value. I'd make that change if I were you; I know how much @Jojo-Schmitz hates compiler warnings ;)